### ISSUE TITLE
Has this project been abandoned?

### ISSUE THREAD

**Issue by mdxprograms** [voltrb/volt#356]
I haven't seen any commits in a few months just curious if this was still active.

**Comment by jfahrer** [comment]
The project is looking for a new lead developer as @ryanstout does not have the necessary time to work on Volt right now. He announced this on gitter (https://gitter.im/voltrb/volt) a few days ago.

**Comment by mdxprograms** [comment]
Ahh thank you for the update @jfahrer sorry to hear. I really like where this project is/was headed.

**Comment by jfahrer** [comment]
Yes, it would be awesome if somebody could take over!

**Comment by ghost** [comment]
We are a foundation dedicated to the development of free software and technologies and we would like to host this project. @jfahrer @ryanstout @mdxprograms

**Comment by dfyx** [comment]
@Poncho09 do you have any more info on your foundation?

**Comment by ghost** [comment]
Good @dfyx we are one Foundation dedicated to the development of software and technologies free. The foundation has Astian name and our domain is http://astian.org the site is under construction, we are based on the principle of privacy and security for end users and for developers, the latter we are interested in giving you all Possible facilities to perform the coding work.

We have several projects like for example EFrame which is a framework that uses in its core Codeigniter this for PHP.

On the other hand we have made a fork of the discontinued Firefox OS called Astian OS, a virtual assistant among others.

We are interested in hosting the Volt project not only to continue its development, to make it more stable and clear to integrate new features.

Our philosophy is and will always be to handle free licensing, MIT, GPL, AGPL, MPL etc.

**Comment by kofronpi** [comment]
Volt is a big project. I don't mean to disrespect but your repos don't have many stars :)

**Comment by brejoc** [comment]
@Poncho09 Any progress here? Btw, how could the foundation help Volt? Would there be financial help involved, so that @ryanstout could perhaps invest more time again?

**Comment by ghost** [comment]
@brejoc 
Of course, yes, there would be not only financing but also everything necessary to reactivate the project, advertising, hosting the project within the foundation, promotion of the framework, use of the framework in the platforms and services of the foundation etc etc

**Comment by brejoc** [comment]
Well, that sounds good. Thanks @Poncho09! Now there is the question if @ryanstout would be able to put on the hat of the leader again if there would be money involved or if this wouldn't change anything and someone else needs to step up.

**Comment by ryanstout** [comment]
Jumping in here, if I could get paid to work on volt, I would do that.  Also not trying to disrespect @Poncho09 but I'm not sure where any money would come from, it looks like there isn't much to your organization yet.

**Comment by ghost** [comment]
@ryanstout We understand that our website is under construction Ryan but that has nothing to do with the financing and much less with the activity of the projects, everything follows its course especially the main projects. According to our organization we do not see any problem in delivering the financing to this project, but if you see inconveniences that is already out of our reach.

Thank you for your attention.

Stay tuned

**Comment by brejoc** [comment]
Thanks for the clarifications @Poncho09. The foundations website, the only thing we see at the moment, really isn't in best shape. But that has nothing to do with this topic and, as you've pointed out, will change soon. Awesome! Might I suggest, that you two (/cc @ryanstout) get together on a more private channel to discuss the details?
But don't forget to come back here with good news soon. 😉

**Comment by ghost** [comment]
@brejoc I've sent the request to Gmail to talk

**Comment by Sija** [comment]
@Poncho09 u talk with gmail, lol :P

**Comment by ghost** [comment]
Well to see, the lack of interest in our proposal will give up the same until you are willing to talk about it.
A hug, Thanks!

**Comment by mhmenon910** [comment]
@ryanstout Very eagerly waiting for this framework to be active again... ☺👍

**Comment by svoop** [comment]
Same here, would be too sad to see Volt bite the dust.
